extensions/project-launcher/package.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "https://raw.githubusercontent.com/vicinaehq/vicinae/refs/heads/main/extra/schemas/extension.json",
+  "name": "project-launcher",
```

Comment:
this name is too generic, as it only handles a very small subset of IDEs. Consider making it more explicit.